airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -1295,6 +1295,8 @@ class WorkflowRuns(SemiIncrementalMixin, GithubStream):
     API documentation: https://docs.github.com/en/rest/actions/workflow-runs#list-workflow-runs-for-a-repository
     """
 
+    use_cache = True
```

Comment:
what is this for?

Reply:
I added following this recommendation https://docs.airbyte.com/connector-development/cdk-python/http-streams/#nested-streams--caching